fdbserver/storageserver.actor.cpp

```diff
@@ -8517,7 +8519,11 @@ ACTOR Future<Void> update(StorageServer* data, bool* pReceivedUpdate) {
 		data->ssVersionLockLatencyHistogram->sampleSeconds(now() - start);
 
 		start = now();
-		state UpdateEagerReadInfo eager;
+		state bool enableClearRangeEagerReads =
+		    data->storage.getKeyValueStoreType() == KeyValueStoreType::SSD_ROCKSDB_V1
```

Comment:
Can you also add sharded rocks for this?

Reply:
Done.